include/caffe/layers/memory_data_layer.hpp

```diff
@@ -27,6 +27,7 @@ class MemoryDataLayer : public BaseDataLayer<Dtype> {
   virtual inline const char* type() const { return "MemoryData"; }
   virtual inline int ExactNumBottomBlobs() const { return 0; }
   virtual inline int ExactNumTopBlobs() const { return 2; }
+  virtual inline bool ShareInParallel() const { return share_in_parallel_; }
```

Comment:
As in #3588, getters should normally be underscore_case according to the style guide and existing usage.